src/Faker/Provider/zh_TW/Payment.php

```diff
@@ -2,10 +2,20 @@
 
 namespace Faker\Provider\zh_TW;
 
+/**
+ * @deprecated Use {@link \Faker\Provider\Payment} instead
+ * @see \Faker\Provider\Payment
+ */
 class Payment extends \Faker\Provider\Payment
 {
+    /**
+     * @return array
+     * @throws \Exception
```

Comment:
Why? Is there similar doc on the other creditCardDetails methods? If not, we should not add this.

Reply:
There's only the core method and that has a `@return` docblock, but no `@throws` - which I now removed here. Also removed the new docblock tags in `zh_TW\Internet`.

Reply:
@GrahamCampbell can you do final review so we can close?